doc/introduction.rst

```diff
@@ -94,6 +94,8 @@ Copyright and license
 =====================
 
 Portions of khmer are Copyright California Institute of Technology,
-where the exact counting code was first developed; the remainder is
-Copyright Michigan State University.  The code is freely available for
+where the exact counting code was first developed. All other code developed
+through 2014 is copyright Michigan State University. All developed code through
```

Comment:
portions are Copyright Michigan State University and Copyright Regents of the University of California.